README.md

```diff
@@ -14,8 +14,160 @@ Now it's good (see [news from Symfony](http://symfony.com/blog/new-in-symfony-2-
 
 ## Upgrade to Symfony 3
 
-On Symfony 2.8, use the official [Guard component](https://symfony.com/doc/master/cookbook/security/guard-authentication.html) and remove this library from your `composer.json`.
-Then you'll be able to upgrade to Symfony 3.
+On Symfony 2.8, use the official [Guard component](https://symfony.com/doc/master/cookbook/security/guard-authentication.html).
+
+### Step 1 - Remove the library from your composer.json
+
+Be sure to be on Symfony 2.8, open `composer.json` file and remove the library:
+
+Before:
+```json
+{
+    "require": {
+        "php": ">=5.5",
+        "symfony/symfony": "~2.8",
+        ...
+        "knpuniversity/guard-bundle": "~0.1@dev"
+    },
+    "require-dev": {...},
+}
+```
+
+Now:
+```json
+{
+    "require": {
+        "php": ">=5.5",
+        "symfony/symfony": "~2.8",
+        ...
+    },
+    "require-dev": {...},
+}
+```
+
+### Step 2 - Remove it from your AppKernel
+
+Open and remove the Bundle from `AppKernel.php` file:
```

Comment:
I think we can just say this without the code example:

>
If you're using the Symfony framework, remove the KnpUGuardBundle from AppKernel.php.